src/index.ts

```diff
@@ -16,9 +16,15 @@ import {replaceProjectIdToken} from '@google-cloud/projectify';
 import {promisifyAll} from '@google-cloud/promisify';
 import arrify = require('arrify');
 import * as extend from 'extend';
-import {GoogleAuth, CallOptions, grpc as gaxVendoredGrpc} from 'google-gax';
+import {
+  GoogleAuth,
+  CallOptions,
+  grpc as gaxVendoredGrpc,
+  ClientOptions,
+} from 'google-gax';
 import * as gax from 'google-gax';
 import * as protos from '../protos/protos';
+import * as os from 'os';
```

Comment:
looks like this isn't needed?

Reply:
Removed. At this point there may be some automation or a bot we can set up for little changes like this. It could be a real time saver.